scripts/vep/main.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+
+"""
+Use VEP using a dataproc cluster.
+Taken from Matt Welland's script, run_vep_help.py
+
+"""
+
+
+import os
+import hailtop.batch as hb
+from analysis_runner import dataproc
+import click
+
+
+@click.command()
+@click.option('--script', 'script', help='path to VEP main script')
+@click.option('--mt', required=True, help='Hail matrix table to run VEP on')
+def main(script: str, mt: str):
+    """
+    runs a script inside dataproc to execute VEP
+    :param script: str, the path to the VEP main script
+    """
+
+    service_backend = hb.ServiceBackend(
+        billing_project=os.getenv('HAIL_BILLING_PROJECT'),
+        bucket=os.getenv('HAIL_BUCKET'),
+    )
+
+    # create a hail batch
+    batch = hb.Batch(name='run_vep_in_dataproc_cluster', backend=service_backend)
+
+    job = dataproc.hail_dataproc_job(
+        batch=batch,
+        worker_machine_type='n1-highmem-8',
+        worker_boot_disk_size=200,
+        secondary_worker_boot_disk_size=200,
```

Comment:
I'm not sure, I can test without

Reply:
JFYI, that's what hailclt sets automatically when you start a cluster with `--vep`, and we need it here explicitly because we using a custom initialisation script.

Reply:
Ok, good to know, thanks! @lgruen is it ok if I add this back in?

Reply:
Thanks, @vladsavelyev, and sorry for misleading you, @KatalinaBobowik -- please add it back in.

Reply:
I've got a comment about these workers resources in the VEP setup instructions here: https://github.com/populationgenomics/production-pipelines/tree/from-initial/scripts/vep (this wasn't merged into main though because of change requests which I didn't get back to :-( that's why it's better to merge quickly even if the code is not ideal). I'll be moving these instructions into a new repo for managing scripts for reference data!